src/lib/checksum.lua

```diff
@@ -68,6 +68,62 @@ function verify_packet (buf, len)
    return ipsum(buf+headersize, len-headersize, initial) == 0
 end
 
+local
+function prepare_packet_l4( buf, len, csum_start, csum_off)
```

Comment:
I believe Luke's convention is "local function foo (a, b)".  Note local and function on the same line, and space before the paren, not after.  Also 2-space indent.